src/util/connections.ts

```diff
@@ -93,33 +93,18 @@ export async function addConnection(clusterConnectionTreeProvider: ClusterConnec
     switch (message.command) {
       case 'submit':
         const url = message.isSecure ? (Constants.prefixSecureURL + message.url) : (Constants.prefixURL + message.url);
-        try {
-          await vscode.window.withProgress({
-            location: vscode.ProgressLocation.Notification,
-            title: 'Checking connection...',
-            cancellable: true
-          }, async () => {
-            await Cluster.connect(url, { username: message.username, password: message.password, configProfile: 'wanDevelopment' });
-          });
-
-        } catch (err) {
-          handleConnectionError(err);
-          currentPanel.dispose();
-          addConnection(clusterConnectionTreeProvider, message);
-          break;
-        }
-
         const connection: IConnection = { url, username: message.username, password: message.password, connectionIdentifier: message.connectionIdentifier };
         const connections = getConnections();
         const connectionId = getConnectionId(connection);
         if (connections && connections[connectionId]) {
-          const answer = await vscode.window.showInformationMessage(`A connection to this cluser has already been established with the name '${connections[connectionId].connectionIdentifier}'.\n Would you like to replace it with a new connection named  '${connection.connectionIdentifier}'`, { modal: true }, "Replace", "Connect to Existing");
+          const answer = await vscode.window.showInformationMessage(`A connection name '${connections[connectionId].connectionIdentifier}' already exist with the same user & server.\n To proceed, either delete the existing connection & try again or continue to the existing connection.`, { modal: true }, "Connect to Existing");
```

Comment:
Could you please change it to: "The connection named 'xxxxx' already contains the same user and server. To proceed, either delete the existing connection & try again or continue to the existing connection"